### DISCUSSION TITLE
Tailwind JIT

### DISCUSSION THREAD

**Discussion by tshafer** [spatie/laravel-support-bubble#1] [Bugs] [answered]
If you are using JIT with tailwind you need to add `/vendor/spatie/laravel-support-bubble/config/**/*.php` to the purge section in tailwind config, perhaps add this to docs?

**Comment by freekmurze** [comment, marked as answer]
@tshafer Feel free to PR this!

**Comment by abishekrsrikaanth** [comment]
This is mentioned on the docs https://github.com/spatie/laravel-support-bubble#include-tailwindcss

**Comment by duongtdvn** [comment]
I am not using JIT, however in production those lines should be added too. Otherwise you will not find the bubble anywhere!

**Comment by AlejandroAkbal** [comment]
If you modify the config file with other classes, you will need to add this line too:

```js
'./config/support-bubble.php',
```

**Reply by lucidpolygon** [reply]
Thanks, my friend. This helped :)
